components/input-number/InputNumber.razor.cs

```diff
@@ -35,7 +35,7 @@ public TValue Step
             {
                 _step = value;
                 NumberFormatInfo nfi = CultureInfo.NumberFormat;
-                var stepStr = _step.ToString();
+                var stepStr = Convert.ToDecimal(_step).ToString();
```

Comment:
Maybe we can add NumberFormatInfo to `ToString()`?

```
var stepStr = Convert.ToDecimal(_step).ToString(nfi);
```

Reply:
I don't think it's necessary to include it, because the ToDecimal will always convert the value into the desired format, and ToString gives the string form of the value without any difficulty.
Should we add it for something else?